### ISSUE TITLE
Fix missing AMA cases after case is put on hold

### ISSUE THREAD

**Issue by yoomlam** [department-of-veterans-affairs/caseflow#13664]
## Description
After a judge puts an AMA case on hold, when they return to their queue view (http://localhost:3000/queue), AMA cases are no longer listed until the page is reloaded/refreshed.

Video: https://www.loom.com/share/f1a89f9e914f405fa42bd113b5d8edec
* Queue starts with 29 cases
* After AMA case is put on hold, 17 cases are listed, none of them are AMA cases.
* After manual refresh, 28 cases are listed. The 1 case put on hold should not be listed.
  * Ticket #10044 will show an "On hold" tab with on-hold cases.

## Acceptance criteria
- [ ] AMA cases are present (without a manual page refresh) in the judges queue view after the judge puts a case on hold

## Background/context/resources
https://dsva.slack.com/archives/CHX8FMP28/p1583866360253300

## Technical notes
Legacy cases cannot be put on hold, at least in these views.

**Comment by yoomlam** [comment]
Somewhat related to #8095.

**Comment by lomky** [comment]
_[what is this chart?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | 
2 | |||||
3 | ||||||||
5 | 
8 | 
```

full reload gets json, after the task is put on hold its from redux. Somethign is improperly resetting the task list without the AMA tasks.

Why 2?
- Might be accidentally clearing ama tasks somewhere

Why 3?
- Because frontend, redux store, ...
- Time to investigate root cause

**Comment by hschallhorn** [comment]
😕  Having a hard time reproducing this again. Steps I followed:
1. Log in as bvaaabshire
1. Go to their queue
1. Click an ama case
1. Select place task on hold from the task action dropdown
1. Submit the hold
1. Click back to your queue
1. Ensure all ama cases appear in queue

Also tested searching for a case so that tasks were not loaded through queue before placing a task on hold. Looks like everything loads correctly.

Thoughts:
This may have been resolved by #14252. One of the changes was that every time queue is loaded, we pull tasks from the back end.
